core/test/FixedPoint.js

```diff
@@ -184,6 +214,26 @@ contract("FixedPoint", function(accounts) {
     assert(await didContractThrow(fixedPoint.wrapDiv("1", "0")));
   });
 
+  it("Division, with ceil", async function() {
```

Comment:
As discussed IRL, a test case that I think would be interesting: 1e-18/1e36 should equal 1e-18 when ceiled.
```
await fixedPoint.wrapDivCeil("1", web3.utils.toWei(web3.utils.toWei("1"))) -> "1"
```